src/child/containers/sidebars/search/Search.js

```diff
@@ -64,11 +65,29 @@ class Search extends Component {
                 </div>
                 <div id="search-scroll" ref="searchscroll" className="side-scroll custom-scrollbar">
                     <div className="sidetab hiddenOnContracted">
-                        {hasErrors && <div className="results-message">
-                            An error occurred while retrieving data. Please check your internet connection or wait for our data services to be re-established.
-                        </div>
-}
-                        {isSearching && <div className="loading-message results-message">Loading search results...</div>}
+                        {hasErrors &&
+                            <div className="results-message">
+                                An error occurred while retrieving data. Please check your internet connection or wait for our data services to be re-established.
+                            </div>
+                        }
+
+                        {isSearching &&<div className="loading-message results-message">Loading search results...</div>}
+
+                        {!isSearching && !results && favourites && favourites.length > 0 && (favourites || []).map(stockCode =>
+                            <Favourite
+                                key={stockCode}
+                                stockCode={stockCode}
+                                bindings={bindings}
+                                selected={stockCode === selection.code}
+                                isFavourite={favourites.indexOf(stockCode) >= 0}
```

Comment:
We don't need this prop, the current Favourite propTypes are wrong. We should remove the isFavourite prop from the Favourite component propTypes and then remove the usage in Favourites.js. Looks like we'd based it on the search prop types at some point.

Also the favourites should get shown in the style of a search result (Just code, name and star), when shown here.